package/MDAnalysis/analysis/contacts.py

```diff
@@ -520,6 +279,8 @@ def __init__(self, *args, **kwargs):
         # - select contact pairs to write out as a timecourse
         # - make this selection based on qavg
         from os.path import splitext
+        
+        warnings.warn("class ContactAnalysis1 will be deprecated, use Contacts instead", DeprecationWarning)
```

Comment:
Very good to deprecate CA1. Just change the string to read "ContactAnalysis1 is deprecated and will be removed in 1.0/. Use Contacts instead.

If there's a doc string, add the 
```rest
.. deprecated: 0.14.0
```

markup (or whatever version this goes in).